pkgs/applications/editors/your-editor/default.nix

```
@@ -1,14 +1,14 @@
-{ lib, stdenv, fetchFromGitHub }:
+{ lib, gccStdenv, fetchFromGitHub }:
 
-stdenv.mkDerivation rec {
+gccStdenv.mkDerivation rec {
   pname = "your-editor";
-  version = "1206";
+  version = "1303";
 
   src = fetchFromGitHub {
-    owner = "kammerdienerb";
+    owner = "your-editor";
     repo = "yed";
-    rev = "6cdd99fe1359899b26d8967bd376fd5caa5451eb";
-    sha256 = "0XECSolW/xPXd1v3sv9HbJMWuHGnwCOwmHoPNCUsE+w=";
+    rev = "1b044e5704fcf8de142ae1d5a6eb45728f0d91a2";
```

releases seems to be tagged now
```suggestion
    rev = version;
```